azurerm/internal/services/recoveryservices/tests/data_source_backup_policy_vm_test.go

```diff
@@ -36,7 +36,7 @@ func testAccDataSourceBackupPolicyVm_basic(data acceptance.TestData) string {
 
 data "azurerm_backup_policy_vm" "test" {
   name                = azurerm_backup_policy_vm.test.name
-  recovery_vault_name = azurerm_backup_vault.test.name
+  recovery_vault_name = azurerm_backup_policy_vm.test.name
```

Comment:
This gets the correct string, but by the incorrect reference, I think it should be:
```suggestion
  recovery_vault_name = azurerm_recovery_services_vault.test.name
```